### PR TITLE
tbi(@sap-ux/ui5-info): Adds support for UI5 themes

### PR DESCRIPTION
**Pull request by IainSAP**
https://github.com/SAP/open-ux-tools/issues/1676

### Description (include screenshots)

- Adds UI5 theme support
- Adds tests to cover
- Renames some files for consistency
- Splits ui5 version and ui5 theme functions and tests to separate files

### Value
- Required for modules soon to be added to `open-ux-tools`

### CODE REVIEW

File: packages/ui5-info/src/ui5-theme-info.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+const MIN_UI5_VER_DARK_THEME = '1.72.0';
+const MIN_UI5_VER_HORIZON_THEME = '1.102.0';
+
+export const enum ui5ThemeIds {
+    SAP_BELIZE = 'sap_belize',
+    SAP_FIORI_3 = 'sap_fiori_3',
+    SAP_FIORI_3_DARK = 'sap_fiori_3_dark',
+    SAP_HORIZON = 'sap_horizon',
+    SAP_HORIZON_DARK = 'sap_horizon_dark'
+    // SAP_HORIZON_CONTRAST = 'sap_horizon_hcw',
```
Comment (cianmSAP) [inline packages/ui5-info/src/ui5-theme-info.ts:14]: Just curious why are these commented out?

**Review by cianmSAP — DISMISSED** [review]
Changeset ✅
Code changes look good
Coverage ✅

File: packages/ui5-info/src/ui5-theme-info.ts (hunk 1 above)
Comment (IainSAP) [reply packages/ui5-info/src/ui5-theme-info.ts:14]: They were commented out on the internal versions...so I dont know. But Ill remove them now, thanks @cianmSAP .

File: packages/ui5-info/src/ui5-version-info.ts [hunk 2]
```diff
@@ -379,3 +380,17 @@ export async function getUI5Versions(filterOptions?: UI5VersionFilterOptions): P
...
 }
+
+/**
+ * Filters an array of versions based on a ui5 version parameter and returns a boolean for code assist libraries enablement.
+ *
+ * @param versions - array of versions
+ * @param ui5Version - minimum version to filter
+ * @returns true if the specified ui5 version supports code assist
+ */
+export function uI5VersionsWithCodeAssist(versions: string[], ui5Version: string): boolean {
```
Comment (IainSAP) [inline packages/ui5-info/src/ui5-version-info.ts:391]: Remove

**Review by cianmSAP — DISMISSED** [review]
Same comments as before, comment resolved 👍

**Review by cianmSAP — APPROVED** [review]

**Review by Klaus-Keller — APPROVED** [review]
Thanks @IainSAP,
- changes to add theming info looks good
- test coverage is great
- did no local test

Approved from my side

**Pull request merged**